src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.helpers.ts

```diff
@@ -0,0 +1,77 @@
+import moment from "moment";
+import { FetchStep, FlowOutcome, FlowStatus, FlowSummaryDataFragment } from "src/app/api/kamu.graphql.interface";
+import AppValues from "src/app/common/app.values";
+
+export class DatasetFlowTableHelpers {
+    public static descriptionColumnTableOptions(element: FlowSummaryDataFragment): { icon: string; class: string } {
```

Comment:
The functions in this class are very hard to reed.
I tried to convert it mostly to `switch-case`, and noticed several missed states.
Added some basic support or TODO comments.

Reply:
+